yoko-testify/src/test/java/testify/hex/HexBuilderTest.java

```diff
@@ -0,0 +1,151 @@
+package testify.hex;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Expand the wildcards in the imports so they are all spelled out.

Reply:
Solved